go/service/rekey.go

```diff
@@ -111,7 +117,66 @@ func (h *RekeyHandler) DebugShowRekeyStatus(ctx context.Context, sessionID int)
 }
 
 func (h *RekeyHandler) RekeyStatusFinish(ctx context.Context, sessionID int) (keybase1.Outcome, error) {
-	return h.gregor.RekeyStatusFinish(ctx, sessionID)
+	outcome, err := h.gregor.RekeyStatusFinish(ctx, sessionID)
+	if err == nil {
+		// recheck rekey status 24h from now
+		h.recheckMu.Lock()
+		h.recheckDeadline = time.Now().Add(24 * time.Hour)
+		h.recheckMu.Unlock()
+	}
+	return outcome, err
+}
+
+func (h *RekeyHandler) recheckRekeyStatusPeriodic() {
+	h.G().Log.Debug("starting recheck rekey status loop")
+	ticker := time.NewTicker(1 * time.Hour)
```

Comment:
Can we use the clock in `G` so that we can eventually try to write tests for this stuff?